tests/unit/test_koji_tag.py

```diff
@@ -206,3 +206,23 @@ def test_invalid_builds_by_attribute_and_label(sample_tagged_builds):
         kt.builds_by_attribute_and_label('farkle',
                                          'name',
                                          'my-project-selinux')
+
+
+def test_str_returns_tag():
+    """
+    GIVEN we have a KojiTag object
+    WHEN we call through to the __str__() method
+    THEN we get back a string with the tag it was based on
+    """
+    kt = build_tag('foo')
+    assert 'foo' == str(kt)
+
+
+def test_repr_returns_tag():
+    """
+    GIVEN we have a KojiTag object
+    WHEN we call through to the __repr__() method
+    THEN we get back a string and throw no exceptions
+    """
+    kt = build_tag('foo')
+    kt.__repr__()
```

Comment:
Wouldn't calling repr(kt) directly generally be more readable and consistent with calling str() in the other test? Also, should we assert something here, even if maybe just something simple like "KojiTag" being part of the string? If we don't think that'd be valuable, that's cool too!

Reply:
it would be more readable and let me adjust the unittest to see if I can get some additional assert

Reply:
adjusted, good now?